wire-ios-data-model/Source/Utilis/EAR/EARService.swift

```diff
@@ -168,6 +168,16 @@ public class EARService: EARServiceInterface {
         }
     }
 
+    private var isEAREnabled: Bool {
```

Comment:
I would see it more as a function as it is not cheap to do (perform on all contexts)

Reply:
done